pennylane/devices/default_qutrit_mixed.py

```diff
@@ -231,15 +231,18 @@ def f(x):
 
         .. note::
             Relaxation error, defined by `readout_relaxation_probs` and implemented with
-            class:`~.QutritAmplitudeDamping`, is applied first. Then misclassification, defined
-            by `readout_misclassification_probs` and implemented with :class:` `~qml.TritFlip`,
-            is applied.
+            class:`~.QutritAmplitudeDamping`, is applied to every measured wire. After that
+            misclassification error, defined by `readout_misclassification_probs` and implemented
+            with :class:` `~qml.TritFlip`, is then applied to each measured wire.
```

Comment:
```suggestion
            :class:`~.QutritAmplitudeDamping`, is applied to every measured wire. After that,
            misclassification error, defined by `readout_misclassification_probs` and implemented
            with :class:`~.TritFlip`, is then applied to each measured wire.
```

Reply:
Could you please double check that this renders correctly?

Reply:
It looks good to me, but I'm not sure about the two notes in the Readout error section should I just make them one "note" but labelled one two: render docs [here](https://xanaduai-pennylane--5842.com.readthedocs.build/en/5842/code/api/pennylane.devices.default_qutrit_mixed.DefaultQutritMixed.html)

Reply:
The content of the first note could be streamlined (to have less repetition with the previous bullet points), and moved into the main text, rather than being separated into a box.